todowebapp/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/checkmark-square.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TODO app</title>
+    <link href="https://fonts.googleapis.com/css?family=Inter" rel="stylesheet">
+</head>
+  <body>
+    <div id="root"></div>
```

Comment:
One other thing you could do is:

```suggestion
    <main></main>
```

Thinking about it: using `<main>` inside a component, which is ostensibly meant to be encapsulated and reusable, kind of breaks that encapsulation. [A document mustn't have more than one `<main>` element that doesn't have the hidden attribute specified.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main#:~:text=A%20document%20mustn%27t%20have%20more%20than%20one%20%3Cmain%3E%20element%20that%20doesn%27t%20have%20the%20hidden%20attribute%20specified.)

As such, we move `main` up into `index.html`, and just use `main` as our application root. 

We'll adjust our CSS slightly, since we no longer want to target `#main` with it (because of encapsulation).

Additionally, since `<main>` elements are guaranteed to be unique, we don't need the `id` anymore--it's redundant.

Related code changes are below, for your convenience.